tests/template_tests/syntax_tests/test_basic.py

```diff
@@ -346,6 +346,52 @@ def test_ignores_strings_that_look_like_format_interpolation(self):
         output = self.engine.render_to_string("tpl-weird-percent")
         self.assertEqual(output, "% %s")
 
+    @setup(
+        {"template": "{{ class_var.class_property }} | {{ class_var.class_method }}"}
+    )
+    def test_subscriptable_class(self):
+        class MyClass(list):
+            # As of Python 3.9 list defines __class_getitem__ which makes it
+            # subscriptable.
+            class_property = "Example property"
+            do_not_call_in_templates = True
+
+            @classmethod
+            def class_method(cls):
+                return "Example method"
+
+        for case in (MyClass, lambda: MyClass):
```

Comment:
Is it not significant that the callable has `do_not_call_in_templates = False`?
```suggestion

            @classmethod
            def class_method(cls):
                return "Example method"

        class MyClassNoCallInTemplates(MyClass):
            do_not_call_in_templates = True

        for case in (MyClassNoCallInTemplates, lambda: MyClass):
            with self.subTest(case=case):
```

Reply:
Yeah, I thought that, too. But the callable is the lambda (which does not have `do_not_call_in_templates` set ;-) ).

Reply:
Ahhhhhh magic :magic_wand: